pkg/pillar/types/zedmanagertypes.go

```diff
@@ -282,3 +339,53 @@ type SignatureInfo struct {
 	SignerCertPem        []byte
 	Signature            []byte
 }
+
+func locationFromDir(locationDir string) (string, error) {
```

Comment:
Ditto; footypes.go are for things related to the type which the producers or consumers of the type share. Not random selections of functions which are used by only one agent.

Reply:
Agreed.. Moved these to DomainMgr and made it a local fn.